apps/src/lib/kits/maker/ui/SetupGuide.jsx

```diff
@@ -337,10 +341,19 @@ const SetupInstructions = () => (
 const MAKER_SETUP_PAGE_URL = document.location.origin + '/maker/setup';
 
 class ChromebookInstructions extends React.Component {
-  render() {
+  webSerialSetupInstructions() {
+    return (
+      <div>
+        {applabI18n.makerSetupChromebook()}
+        <h4> Note </h4>
```

Comment:
we should probably pull this string from i18n. 

also, @mgc1194 is currently working on moving the rest of this page to react and making everything translatable -- mario, just tagging you here so you can incorporate these changes into your branch once they've been merged into staging